grue.go

```diff
@@ -29,6 +31,42 @@ func add(args []string, conf *config.GrueConfig) error {
 	return conf.AddAccount(name, uri)
 }
 
+func list(args []string, conf *config.GrueConfig) {
+	const (
+		fmtShort = "%s\t%s\n"
+		fmtFull  = "%s:\n%s\n"
+	)
+	var full bool
+	var listCmd = flag.NewFlagSet("list", flag.ExitOnError)
+	listCmd.BoolVar(&full, "full", false, "Show full account info")
+	listCmd.Parse(args)
+	if len(listCmd.Args()) == 0 {
+		if full {
+			for name, cfg := range conf.Accounts {
+				fmt.Printf(fmtFull, name, cfg.String())
+			}
+			return
+		}
+		w := tabwriter.NewWriter(os.Stdout, 0, 8, 0, '\t', 0)
+		for name, cfg := range conf.Accounts {
+			fmt.Fprintf(w, fmtShort, name, cfg.URI)
+		}
+		w.Flush()
+		return
+	}
+
+	args = listCmd.Args()
+	name := args[0]
+	listCmd.Parse(args[1:])
```

Comment:
Lines 58-60 can just be `name := listCmd.Args()[0]`
You've already parsed out the flag for -full so the name is the only argument left
Also makes it so you could `grue list -full <name>` instead of only allowing

Reply:
I was trying to allow both `grue list -full <name>` and `grue list <name> -full`. Will change to only support the former.